### DISCUSSION TITLE
How to use broadcast to receive messages triggered by three-party hardware in maui

### DISCUSSION THREAD

**Discussion by HuaFangYun** [dotnet/maui#13468] [Ideas]
### Description

How to use broadcast to receive messages triggered by three-party hardware in maui

### Public API Changes

no

### Intended Use-Case

Infrared QR code scanning of handheld devices. Press the scan button of the handheld device, and the app receives the message.

**Comment by acaliaro** [comment]
You should take a look to the device documentation. Usually, on Android devices, the OS use Intents to send the barcode to the apps. Which is the device vendor?

**Comment by HuaFangYun** [comment]
Thank you for your reply. I have the equipment document here. The original file is jar. Maui binding is required, but then broadcast message registration and reception are required. But I don't know how to broadcast and receive these. Do you have a sample program here?

**Reply by acaliaro** [reply]
which is the device vendor?

**Reply by HuaFangYun** [reply]
Small suppliers of equipment in China. Message broadcasting is used in the java code for device docking to receive data. Please tell me how maui receives broadcast messages. Is there an example code?

**Comment by saber-wang** [comment]
There are almost all objects in the Android SDK in MAUI, so you only need to refer to the Java example and implement it in 'MainActivity'

**Comment by HuaFangYun** [comment]
Thank you. The problem has been solved.

**Reply by Giant150** [reply]
I also ran into this issue, can you tell me how it was solved?
